tools/system_libs.py

```diff
@@ -733,19 +733,6 @@ def get_files(self):
         # TODO: These could be moved away from JS in the upcoming musl upgrade.
         'pthread_cancel.c', 'pthread_detach.c',
         'pthread_join.c', 'pthread_testcancel.c',
-        # TODO: Support C11 condition variable and mutex library functions.
-        'cnd_broadcast.c',
-        'cnd_destroy.c',
-        'cnd_init.c',
-        'cnd_signal.c',
-        'cnd_timedwait.c',
-        'cnd_wait.c',
-        'mtx_destroy.c',
-        'mtx_init.c',
-        'mtx_lock.c',
-        'mtx_timedlock.c',
-        'mtx_trylock.c',
-        'mtx_unlock.c',
```

Comment:
Do you think we should also include these in the stub implementation (so that it fails at runtime but still works during compilation, when not targeting pthreads)?

Reply:
Ah yes, forgot to do that.    I need to figure out where we test for that..

Reply:
Added, and improved tests for stub functions.